docs/source/entities-advanced.mdx

```diff
@@ -87,7 +87,24 @@ type Product @key(fields: "id", resolvable: false) {
 type Product @key(fields: "sku", resolvable: false) {
   sku: String!
 }
+```
+
+When resolving a reference for an entity with multiple keys, you can determine how to resolve it based on which key is present:
```

Comment:
```suggestion
When resolving a reference for an entity with multiple keys, you can determine how to resolve it based on which key is present. For example, if you're using [`@apollo/subgraph`](/apollo-server/using-federation/api/apollo-subgraph/), it could look like this:
```